### DISCUSSION TITLE
Upgrade to v.2.0.0

### DISCUSSION THREAD

**Discussion by koleto** [quasarframework/quasar#15315] [General - Components / Directives / etc] [answered]
I am somewhat confused by the upgrade to v.2.0.0. For about 2 Months I did create new projects with the current cli + vite plugin. I also keep them up-to-date by executing `quasar upgrade -i`. When I execute `quasar dev` the log displays:
```
 » Dev mode............... spa
 » Pkg quasar............. v2.11.5
 » Pkg @quasar/app-vite... v1.2.0
 » Browser target......... es2019|edge88|firefox78|chrome87|safari13.1
```
Outside the project directory, I have:
```
quasar -v
2.0.0
```

Inside the project directory, I have:
```
 quasar -v
1.2.0
```
I am using `Vue 3`+`Vite` inside the projects in a generated quasar app.

So am I up to date and if not what do I have to do to keep the projects to the last version?

**Comment by smolinari** [comment, marked as answer]
Hey. 

So, when you are using the Quasar CLI, you aren't using the "vite-plugin". You are using the Vite version of the CLI app. There is also a webpack version, which is the legacy version. The vite-plugin is for using Quasar inside a purely Vue+Vite driven app i.e. without the Quasar CLI/ using only the Vue-Vite CLI. 

So, when inside of a Quasar app and using the Quasar CLI as you are doing, the CLI "hooks" into "app-vite" to get further commands to be able to work with your project. Outside of your Quasar project, you are working only with the global Quasar CLI. 

So, looking at your global CLI version, 2.0.0 is the most up-to-date version. 

Looking inside your project, your app-vite version, 1.2.0 is also the most up-to-date version. 

So, you are up-to-date. :)

You can check the latest versions here:

![image](https://user-images.githubusercontent.com/3314957/214264532-27adfb7c-6190-41de-aa5a-5a9f780a17e2.png)

Go into each package and look at the package.json for the current version.

Hope that makes the parts of Quasar a bit more understandable. 

Scott

**Reply by koleto** [reply]
Oh Thank you :-) I was not sure!  Mainly I have now the newest version :-)

**Comment by smolinari** [comment]
@koleto - I also just remembered. You can also find the latest version numbers here:

https://quasar.dev/start/release-notes

![image](https://user-images.githubusercontent.com/3314957/214278126-036a604e-c82d-48c5-86cf-80a91a9353a7.png)

Scott

**Reply by koleto** [reply]
Very useful, Thank you!
